pkgs/build-support/cc-wrapper/cc-wrapper.sh

```diff
@@ -91,6 +91,20 @@ if [ "$NIX_ENFORCE_PURITY" = 1 -a -n "$NIX_STORE" ]; then
     params=("${rest[@]}")
 fi
 
+
+# Clear march/mtune=native -- they bring impurity.
+if [ "$NIX_ENFORCE_NO_NATIVE" = 1 ]; then
+    rest=()
+    for i in "${params[@]}"; do
+        if [[ "$i" = -m*=native ]]; then
+            skip $i
+        else
+            rest=("${rest[@]}" "$i")
```

Comment:
Thanks! Fixed in https://github.com/NixOS/nixpkgs/commit/119c287c7177e6abcb43e1eb9e0e9e1b3456cdf7 along with other places like this.